docs/shapes.rst

```diff
@@ -0,0 +1,358 @@
+.. include:: references.txt
+
+.. _sh:
+
+.. _sh-shapes:
+
+Shapes
+======
+
+This section shows one example how to construct a region for each shape that's
+currently supported.
+
+* `~regions.CircleSkyRegion` and `~regions.CirclePixelRegion`
+
+.. code-block:: python
+
+    >>> from astropy.coordinates import Angle, SkyCoord
+    >>> from regions import PixCoord, CircleSkyRegion, CirclePixelRegion
+
+    >>> center_sky = SkyCoord(42, 43, unit='deg', frame='fk5')
+    >>> circle_sky = CircleSkyRegion(center=center_sky, radius=Angle(3, 'deg'))
```

Comment:
Is it necessary to specify these as `Angle` objects, or could one just use `3*u.deg`?  The latter is terser and a little more natural, so I'd prefer it if it's allowed.